extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -142,12 +140,12 @@ private function getUploadErrorMessage( $status ) {
 
 				$msg = wfMsgExt(
 					'filetype-banned-type',
-					array( 'parseinline' ),
+					[ 'parseinline' ],
 					htmlspecialchars( $finalExt ),
 					implode(
 						wfMsgExt(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)